src/js/tracks/text-track.js

```diff
@@ -241,12 +248,10 @@ class TextTrack extends Track {
             // On-demand load.
             loadTrack(this.src, this);
           }
+          this.tech_.off('timeupdate', timeupdateHandler);
```

Comment:
Should we also return early in this function if the mode is the same as the current mode? Is there a reason we trigger `modechange` when a track goes from `disabled` to `disabled` again?

Reply:
Making this change makes sense. I think we'd want to see whether doing that causes any issues for the captions menu, though, I doubt it.